node/silkworm/db/access_layer.cpp

```diff
@@ -194,17 +210,14 @@ bool read_block(mdbx::txn& txn, BlockNum block_number, bool read_senders, BlockW
     SILKWORM_ASSERT(data.value.length() == kHashLength);
     std::memcpy(bh.hash.bytes, data.value.data(), kHashLength);
 
-    // Locate header
-    thread_local mdbx::cursor_managed headers_cursor;
-    headers_cursor.bind(txn, db::open_map(txn, table::kHeaders));
+    // Read header
     key = block_key(block_number, bh.hash.bytes);
-    data = headers_cursor.find(to_slice(key), false);
-    if (!data) {
+    auto raw_header{read_header_raw(txn, key)};
+    if (raw_header.empty()) {
         return false;
     }
-
-    ByteView data_view(from_slice(data.value));
-    rlp::success_or_throw(rlp::decode(data_view, bh.block.header));
+    ByteView raw_header_view{raw_header.data(), raw_header.length()};
```

Comment:
ByteView has a constructor for Bytes, so I think this could be written:
`ByteView raw_header_view{raw_header);`

Reply:
Yes of course ... I think is a matter of taste to use initialization list vs ctor. 
Will change